### DISCUSSION TITLE
Systray to print/change the current major mode?

### DISCUSSION THREAD

**Discussion by tobiasBora** [kmonad/kmonad#844] [Q&A]
TL;DR: Is it possible to put in the systray a module that prints the current layer (I need to switch between different layers like azerty, qwerty, bépo… as multiple users are using my laptop) and let the user change it easily from the systray?

Context: On my laptop I often need to change between three layouts (qwerty international, azerty and bépo) as multiple users are using it. I wanted to use kmonad + ibus (or setxkbmap) to let kmonad deal with shortcuts and ibus deal with the layout… But due to https://github.com/kmonad/kmonad/issues/842 it is not possible to make kmonad compose nicely with `ibus` or `setxkbmap` for instance, meaning that I need to implement everything in kmonad (and understand how to get around https://github.com/kmonad/kmonad/issues/843 but it is another question). But multiple users are using my laptop (KDE plasma), and I want to provide a simple way:
- to see the current layout (aka layer) in the systray
- to change the current layout with a click
as it is already possible in ibus/KDE.

But after a quick search I can't find any way to do that. Am I missing something?
